### PR TITLE
SD 2.1 GPU/Inf Readme's

### PR DESCRIPTION
**Pull request by jonrwong-amazon**
*Issue #, if available:*

*Description of changes:*
Added tutorials on how to launch Stable Diffusion 2.1 onto Nvidia GPU's and AWS Inferentia with EKS. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by yahavb — COMMENTED** [review]
can we use variable for `jon-karpenter` and `tlvsummit`?

File: amd-nvidia-nodepool.yaml
Comment (yahavb) [inline amd-nvidia-nodepool.yaml:1]: same here with exporting labels

File: sd-gpu-readme.MD
Comment (yahavb) [inline sd-gpu-readme.MD:1]: can we add markup notations? titles and file names

**Review by yahavb — COMMENTED** [review]
Looks good few nits

File: amd-nvidia-nodepool.yaml
Comment (jonrwong-amazon) [reply amd-nvidia-nodepool.yaml:1]: yes, updated these files with environment variables

File: sd-gpu-readme.MD
Comment (jonrwong-amazon) [reply sd-gpu-readme.MD:1]: changed both readme's to include headers and a title

**Pull request merged**